<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="com.w2a.testcases.AddCustomerTest">
        <methods>
          <include name="setUp"/>
          <include name="addCustomerTest" invocation-numbers="0 1 2 3"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- com.w2a.testcases.AddCustomerTest -->
      <class name="com.w2a.testcases.BankManagerLoginTest">
        <methods>
          <include name="setUp"/>
          <include name="bankManagerLoginTest"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- com.w2a.testcases.BankManagerLoginTest -->
      <class name="com.w2a.testcases.OpenAccountTest">
        <methods>
          <include name="openAccountTest" invocation-numbers="0"/>
          <include name="setUp"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- com.w2a.testcases.OpenAccountTest -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
